pom.xml

```diff
@@ -15,7 +15,8 @@
   <parent>
     <groupId>org.apache</groupId>
     <artifactId>apache</artifactId>
-    <version>14</version>
+    <version>18</version>
```

Comment:
Never noticed this before. Which version is this? Why does Apache itself have a version? Can we include a comment about this version and the relative path for us newbies?

Reply:
@paul-rogers This is the latest version that majority of Apache projects use (if I remember correctly it was released in May 2016,  version 14 is quite old). Apache parent pom is a maven artifact that defines plugins and other common to ASF project maven parameters and it needs to be versioned. I'd suggest keeping pom file as is, the discussion from the PR is mirrored in the DRILL-5862 and the commit has reference to it.